sdk/eventhub/Microsoft.Azure.EventHubs/src/Primitives/ClientEntity.cs

```diff
@@ -20,6 +20,9 @@ public abstract class ClientEntity
         static int nextId;
         RetryPolicy retryPolicy;
 
+        /// <summary />
+        protected bool closeCalled = false;
```

Comment:
Why is this field protected and named this way?

Reply:
Sorry, missed to notice your comment. I see your point. I will remove it and set the property setter as protected instead in another PR.